Modules/ITS/src/ITSTrackTask.cxx

```diff
@@ -92,6 +111,10 @@ void ITSTrackTask::monitorData(o2::framework::ProcessingContext& ctx)
 
   ILOG(Debug, Devel) << "START DOING QC General" << ENDM;
 
+  //o2::base::GeometryManager::loadGeometry();
+  mGeom = o2::its::GeometryTGeo::Instance();
+  mGeom->fillMatrixCache(o2::math_utils::bit2Mask(o2::math_utils::TransformType::L2G));
```

Comment:
move these two lines inside the `if(mTimestamp == -1)`.

Reply:
In addition here you are using the full geometry. I guess a light-weight version of it should be fine. Please copy what was done for the ClusterTask here: https://github.com/AliceO2Group/QualityControl/pull/2164/files#diff-41e35aa0ad16f0bc0150242f13cb67d78b78f53b020364e33f18cce3fcf44de4R125 

Note also the removal from the json of the `grpGeomRequest`